test/import-map.test.js

```diff
@@ -188,4 +188,54 @@ describe(`/import-map.json`, () => {
     expect(response.body.imports.b).not.toBeDefined();
     expect(response.body.imports["b/"]).not.toBeDefined();
   });
+
+  it(`returns a 404 when you try to retrieve an import map for an environment that doesn't exist`, async () => {
```

Comment:
None of these new tests were passing until i made the corresponding changes to the source.